src/fskit/protection-message.h

```diff
@@ -37,14 +37,9 @@
 #include "griddyn/griddyn-config.h"
 
 #include <string>
-#ifdef GRIDDYN_HAVE_FSKIT
```

Comment:
TODO `#include <boost/serialization/serialization.hpp>` shouldn't've been deleted. This file isn't included unless GRIDDYN_HAVE_FSKIT is set, so this might've failed regardless. (Is <boost/serialization> transative from fskit/event- message?)

Reply:
I think the plan was to convert this message to use cereal instead of boost seriealization like the rest of the commMessages.    I am hoping once HELICS is a little more firmly established we can remove FSKIT completely.    But probably should keep it around for now.  @nightlark  might know more about the current status of this module.

Reply:
The plan was to replace boost serialization with cereal -- I think all of the changes to switch to cereal were made, but maybe I missed moving some of the changes to GitHub from the internal repo.

Reply:
Yeah, this copy of fskit doesn't have any `cereal` text in it at all